pipeline/data_preprocessing/genotype/VCF_QC.ipynb

```diff
@@ -70,16 +70,20 @@
    },
    "source": [
     "## Minimal working example\n",
+    "Following files are required to run the mwe\n",
+    "1. The target vcf file to be QC\n",
+    "2. A reference vcf.gz file to generate the dbsnp_variants\n",
+    "3. A reference fasta file\n",
+    "\n",
     "\n",
     "```\n",
-    "sos run VCF_QC.ipynb dbsnp_annotate \\\n",
-    "    --genoFile data/00-All.vcf.gz\n",
+    "nohup sos run VCF_QC.ipynb dbsnp_annotate \\\n",
```

Comment:
as discussed, please dont put `nohup` in notebooks from now on. If you want you can put in your bashrc something like `alias sos=nohup sos`